pipeline/src/types/transformed/index.ts

```diff
@@ -69,6 +69,7 @@ export type ElasticsearchArticle = {
 
 export type ElasticsearchEventDocument = {
   id: string;
+  childScheduledEvent?: boolean;
```

Comment:
Could we rename to `isChildScheduledEvent` as we try to do that for booleans (`has`, `is`, type prefixes)